pages/_app.js

```diff
@@ -1,10 +1,27 @@
 import GlobalStyle from "../styles";
+import { useState } from "react";
 
 export default function App({ Component, pageProps }) {
+  const [color, setColor] = useState({
+    color1: "#C33737",
+    color2: "#4287A9",
+    color3: "#2F933A",
+  });
+  const handleColorChange = (id, value) => {
+    setColor((prevColor) => ({
+      ...prevColor,
+      [id]: value,
+    }));
+  };
```

Comment:
Please use the function declaration instead of the arrow function. Not because it is wrong, but because it is a convention in our curriculum and we want a uniform spelling for you and for us.🙂